platform/lib/routers/pages.js

```diff
@@ -165,7 +165,7 @@ pages.get('/*', async (req, res, next) => {
   // Let known file extensions automatically fallthrough as if they could not
   // been resolved by the preceeding middleware the pages router can't
   // resolve them either
-  const KNOWN_FILE_EXTENSIONS = /\.(jpg|png|css|js|map)$/;
+  const KNOWN_FILE_EXTENSIONS = /\.(jpg|png|css|js|xml|map)$/;
```

Comment:
Maintaining an explicit filter feels a bit fragile. Wouldn't it make more sense to check if there's no file extension or `.html`?

Reply:
//cc @matthiasrohmer

Reply:
Jup, totally fine to turn it around to check for `/`, `$` and `.html` instead!

Reply:
Implemented a matching pattern, so that the handler method is only called when the path does not point to a file with an extension other than html